packages/flutter/test/cupertino/activity_indicator_test.dart

```diff
@@ -79,7 +136,8 @@ void main() {
           key: key,
           child: Container(
             color: CupertinoColors.white,
-            child: const CupertinoActivityIndicator.partiallyRevealed(progress: 0),
```

Comment:
Nitpick: Not sure what triggered these changes. Have the Flutter formatting rules changed, or is this just based on  the auto-formatting preferences from whatever IDE the PR author is using?  There's a bunch of formatting changes in this PR that mask the actual changes under the covers.

Reply:
The file is formated using dartfmt. I think the diff tool should be able to show the actual change in a darker color.